pages/docs/couple-your-code/couple-your-code-defining-mesh-connectivity.md

```diff
@@ -95,3 +95,108 @@ void setMeshTriangles(int meshID, int size, int* vertices);
 void setMeshQuads(int meshID, int size, int* vertices);
 void setMeshTetrahedra(int meshID, int size, int* vertices);
 ```
+
+## Putting it all together
+
+Solvers may give you a range of information regarding vertices and faces.
+Some may give you unique ids for vertices, some only coordinates.
+In this section, we handle some common cases and how to implement them.
+
+### Solver provides IDs
+
+You solver gives each vertex or node a unique identifier.
```

Comment:
```suggestion
Your solver gives each vertex or node a unique identifier.
```

also: why distinguishing between vertices and nodes? I understand the same thing. This also applied below.

Reply:
Hmm, I don't destinguish between the two. Both are common names, so I mention both. Thinking about it, face centers are also common...

Reply:
I changed the wording to `vertex` only